etc/config/python.amazon.properties

```diff
@@ -58,3 +58,20 @@ group.pythran.baseName=Pythran
 compiler.pythran015.exe=/opt/compiler-explorer/pythran/pythran-0.15/bin/pythran
 compiler.pythran015.semver=0.15
 compiler.pythran015.cpp_compiler_root=/opt/compiler-explorer/gcc-13.2.0/
+
+## Codon
+group.codon.compilers=codon0192
+group.codon.interpreted=false
+group.codon.compilerType=codon
+group.codon.licenseLink=https://raw.githubusercontent.com/exaloop/codon/refs/heads/develop/LICENSE
+group.codon.licenseName=Apache License 2.0
+
+group.codon.supportsBinary=false
+group.codon.supportsBinaryObject=false
+group.codon.supportsExecute=false
+group.codon.objdumper=/opt/compiler-explorer/gcc-13.2.0/bin/objdump
+group.codon.isSemVer=true
+group.codon.baseName=Codon
+
+compiler.codon0192.exe=/opt/compiler-explorer/codon/codon-0.19.2/bin/codon
```

Comment:
This isn't the right path - the installer you added installed it here:

```
Aug/11 15:42 admin-node~/infra (main|✔) 6.0s $ ls /opt/compiler-explorer/codon-0.19.2/bin/codon 
/opt/compiler-explorer/codon-0.19.2/bin/codon
```

We can move it in the installer, or fix the path here. Up to you :)

Reply:
Fixed - I changed the path here.